.gopherci.yml

```diff
@@ -0,0 +1,4 @@
+# xorg-dev and libgl1-mesa-dev are cgo dependencies for glfw (imported by dumpglfw3joysticks).
```

Comment:
Get rid of the comment, I don't think it's necessary. I prefer to find this information via git blame and the commit message.